GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,13 +8,145 @@
+                cell.firstCellNameLabel.text = season3[indexPath.row].name
+                cell.firstCellEpisodeLabel.text = "S\(season3[indexPath.row].season) Ep \(season3[indexPath.row].number)"
+                return cell
+            }
+        case 3:
+            if let cell = tableOutlet.dequeueReusableCell(withIdentifier: "SecondCell") as? SecondTableViewCell {
+                cell.secondCellPicOutlet.image = UIImage(named: season4[indexPath.row].originalImageID)
+                cell.secondCellNameLabel.text = season4[indexPath.row].name
+                cell.secondCellEpisodeLabel.text = "S\(season4[indexPath.row].season) Ep \(season4[indexPath.row].number)"
+                return cell
+            }
+        case 4:
+            if let cell = tableOutlet.dequeueReusableCell(withIdentifier: "FirstCell") as? FirstTableViewCell {
+                cell.firstCellPicOutlet.image = UIImage(named: season5[indexPath.row].originalImageID)
+                cell.firstCellNameLabel.text = season5[indexPath.row].name
+                cell.firstCellEpisodeLabel.text = "S\(season5[indexPath.row].season) Ep \(season5[indexPath.row].number)"
+                return cell
+            }
+        case 5:
+            if let cell = tableOutlet.dequeueReusableCell(withIdentifier: "SecondCell") as? SecondTableViewCell {
+                cell.secondCellPicOutlet.image = UIImage(named: season6[indexPath.row].originalImageID)
+                cell.secondCellNameLabel.text = season6[indexPath.row].name
+                cell.secondCellEpisodeLabel.text = "S\(season6[indexPath.row].season) Ep \(season6[indexPath.row].number)"
+                return cell
+            }
+        case 6:
+            if let cell = tableOutlet.dequeueReusableCell(withIdentifier: "FirstCell") as? FirstTableViewCell {
+                cell.firstCellPicOutlet.image = UIImage(named: season7[indexPath.row].originalImageID)
+                cell.firstCellNameLabel.text = season7[indexPath.row].name
+                cell.firstCellEpisodeLabel.text = "S\(season7[indexPath.row].season) Ep \(season7[indexPath.row].number)"
+                return cell
+            }
+        default:
+            return UITableViewCell()
+        }
+        return UITableViewCell()
+    }
+    
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
+    }
+
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return "Season \(section + 1)"
+    }
+    
 
+    @IBOutlet weak var tableOutlet: UITableView!
+    
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let segueIdentifier = segue.identifier else {fatalError("No identifier in segue")}
+
+
+        switch segueIdentifier {
+        case "listToDetailSegue":
+            guard let detailVC = segue.destination as? DetailViewController else { fatalError("Unexpected segue view controller" )}
```

Comment:
In this function using the array of arrays also would have saved you a lot of lines of code.